### ISSUE TITLE
Broken after day light savings time change

### ISSUE THREAD

**Issue by srujanb** [Yelp/beans#122]
Saw the emails messing up time after the day light saving.

**Comment by ymilki** [comment]
Yes, this currently requires manual database changes. We will need to find a better way to handle daylight savings time changes.
